google-cloud-spanner/lib/google/cloud/spanner/pool.rb

```diff
@@ -277,8 +277,7 @@ def can_allocate_more_sessions?
         end
 
         def create_keepalive_task!
-          @keepalive_task = Concurrent::TimerTask.new(execution_interval: 300,
-                                                      timeout_interval: 60) do
+          @keepalive_task = Concurrent::TimerTask.new execution_interval: 300 do
```

Comment:
Why are we removing `timeout_interval` ?

Reply:
timeout_interval is not taken into account by concurrent_ruby now https://github.com/ruby-concurrency/concurrent-ruby/blob/4cfc0a107b10d3ab213396d53411a1ee487200af/lib/concurrent-ruby/concurrent/timer_task.rb#L268 ..  using it puts a lot of unnecessary warnings when running tests

Reply:
Got it, thanks. Can we just add a comment here for that?
Overall LGTM

Reply:
I would prefer adding it in the commit description when merging,
As we are anyways removing the option. Does that sound fine ?